### PR TITLE
Add action limiting

### PR DESCRIPTION
**Pull request by jnordberg**
Adds rate-limiting based on actions taken per ip and user

### CODE REVIEW

**Comment by jnordberg** [comment]
Since the database migrations where not sorted before the following query has to be run prior to running them:

```sql
UPDATE SequelizeMeta SET name='001.create-users.js' WHERE name='create-users.js'
```

File: src/locales/en.json [hunk 1]
```diff
@@ -48,6 +48,7 @@
   "error_api_username_reserved": "Username reserved",
   "error_api_username_used": "This username is already in use",
   "error_api_wait": "Please wait at least one minute between retries",
+  "error_api_actionlimit": "You are doing that too much, please try again tomorrow",
```
Comment (gregory-latinier) [inline src/locales/en.json]: The key is missing in other languages.

Comment (jnordberg) [reply src/locales/en.json]: That should not block this pr (assuming that we fall back to English if another language is not found, right?)

Can you create an issue about adding the translation later?

Comment (gregory-latinier) [reply src/locales/en.json]: If I'm not mistaken react-intl we'll use the key of the message if it's not existing in the locale file

Comment (jnordberg) [reply src/locales/en.json]: That's not good, there should be a fallback locale set

**Pull request merged**